docs/release-notes/2.0.1.md

```diff
@@ -0,0 +1,15 @@
+# Version 2.0.1 (2023-05-)
+
+| Release date |  May , 2023 |
+| ------------ | ------------------------------------------ | 
+| Installation | [Install `pg_stat_monitor`](../install.md) | 
+
+## Release highlights
+
+This release fixes the server crash when running `pgbench` and improves the memory context handling for pg_stat_monitor
```

Comment:
Instead of mentioning that it fixes the server crash when running `pgbench`. I'd replace that with:
This release fixes the server crash when plan was enabled, addresses incorrect behaviour with non-default number of buckets, and improves the memory context handling of pg_stat_monitor.